res/css/views/auth/_LanguageSelector.scss

```diff
@@ -23,6 +23,7 @@ limitations under the License.
     font-size: $font-14px;
     font-weight: 600;
     color: $authpage-lang-color;
+    width: 160px;
```

Comment:
Hmm, I don't think is quite what we want, as it means the menu is no longer centered in the column. If you can fix that up as well, then the larger widget could be okay.